lib/modules/ssh/strong_password/files/ssh_listener.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+import socket
+import sys
+import threading
+import _thread
+import paramiko
+
+
+HOST_KEY = paramiko.RSAKey(filename='/root/.ssh/id_rsa')
+SSH_PORT = 22
+LOGFILE = '/root/logs/logins.txt'  # File to log the user:password combinations to
+LOGFILE_LOCK = threading.Lock()
+
+
+class SSHServerHandler(paramiko.ServerInterface):
+    def __init__(self,client_addr):
+        self.event = threading.Event()
+        self.ip=client_addr[0]
+
+    def check_auth_password(self, username, password):
+        LOGFILE_LOCK.acquire()
+        try:
+            logfile_handle = open(LOGFILE, "a")
+            print("New login : " + username + ":" + password +":" + self.ip + ":" +\
+                  "ssh/strong_password" +"\n")
+            logfile_handle.write(username + ":" + password +":" + self.ip + ":"
```

Comment:
please log the files as json
```
logfile_handle.write(json.dumps({"username": username, "password": password, "IP" : self.ip , "module_name" : "ssh/strong_password"}) + "\n")
```

Reply:
also please date and time into log

Reply:
forget to tell, also log real_machine_identifier_name to know which servers logs are coming from.